code/game/machinery/doors/firedoor.dm

```diff
@@ -442,43 +443,6 @@
 //These are playing merry hell on ZAS.  Sorry fellas :(
 
 /obj/machinery/door/firedoor/border_only
-/*
```

Comment:
I'm not sure why this was all commented out, best leave it as is

Reply:
> I'm not sure why this was all commented out, best leave it as is

Old code there. I think this don't will work.

Reply:
it won't, but leave it there for future reference, we will probably want directional firedoors at some point

Reply:
> it won't, but leave it there for future reference, we will probably want directional firedoors at some point

returned